client/app/services/event-notifications.service.js

```diff
@@ -185,16 +242,22 @@
         if (index > -1) {
           group.notifications.splice(index, 1);
           service.removeToast(notification);
+          if (notification.href) {
+            CollectionsApi.delete('notifications', notification.id);
+          }
           updateUnreadCount(group);
         }
       }
     }
 
     function clearAll(group) {
       if (group) {
+        var resources = [];
         angular.forEach(group.notifications, function(notification) {
```

Comment:
@himdel is it possible to use a simple `map` here? I'm not familiar with this `angular.forEach` and I would like to get rid of the `resources.push` line.